kuma/conftest.py

```diff
@@ -15,6 +15,11 @@
 from kuma.wiki.models import Document, Revision
 
 
+@pytest.fixture(autouse=True)
+def clear_cache():
+    caches["default"].clear()
```

Comment:
This is the more crucial change. This means that any `pytest` based test now starts with an empty cache. It wasn't like this before. Any test that was `unittest` class based based on `KumaTestCase` would always start with a cleared cache.